core/src/main/scala/org/apache/spark/ui/exec/ExecutorsPage.scala

```diff
@@ -54,78 +53,16 @@ private[ui] class ExecutorsPage(
-        for (statusId <- 0 until listener.activeStorageStatusList.size)
-          yield ExecutorsPage.getExecInfo(listener, statusId, isActive = true)
-      }
-      val _deadExecutorInfo = {
-        for (statusId <- 0 until listener.deadStorageStatusList.size)
-          yield ExecutorsPage.getExecInfo(listener, statusId, isActive = false)
-      }
-      (_activeExecutorInfo, _deadExecutorInfo)
-    }
-
-    val execInfo = activeExecutorInfo ++ deadExecutorInfo
-    implicit val idOrder = Ordering[Int].on((s: String) => idStrToInt(s)).reverse
-    val execInfoSorted = execInfo.sortBy(_.id)
-    val logsExist = execInfo.filter(_.executorLogs.nonEmpty).nonEmpty
-
-    val execTable = {
-      <table class={UIUtils.TABLE_CLASS_STRIPED_SORTABLE}>
-        <thead>
-          <th class="sorttable_numeric">Executor ID</th>
-          <th>Address</th>
-          <th>Status</th>
-          <th>RDD Blocks</th>
-          <th><span data-toggle="tooltip" title={ToolTips.STORAGE_MEMORY}>Storage Memory</span></th>
-          <th>Disk Used</th>
-          <th>Cores</th>
-          <th>Active Tasks</th>
-          <th>Failed Tasks</th>
-          <th>Complete Tasks</th>
-          <th>Total Tasks</th>
-          <th><span data-toggle="tooltip" title={ToolTips.TASK_TIME}>Task Time (GC Time)</span></th>
-          <th><span data-toggle="tooltip" title={ToolTips.INPUT}>Input</span></th>
-          <th><span data-toggle="tooltip" title={ToolTips.SHUFFLE_READ}>Shuffle Read</span></th>
-          <th>
-            <!-- Place the shuffle write tooltip on the left (rather than the default position
-              of on top) because the shuffle write column is the last column on the right side and
-              the tooltip is wider than the column, so it doesn't fit on top. -->
-            <span data-toggle="tooltip" data-placement="left" title={ToolTips.SHUFFLE_WRITE}>
-              Shuffle Write
-            </span>
-          </th>
-          {if (logsExist) <th class="sorttable_nosort">Logs</th> else Seq.empty}
-          {if (threadDumpEnabled) <th class="sorttable_nosort">Thread Dump</th> else Seq.empty}
-        </thead>
-        <tbody>
-          {execInfoSorted.map(execRow(_, logsExist))}
-        </tbody>
-      </table>
-    }
-
     val content =
-      <div class="row">
-        <div class="span12">
-          <h4>Summary</h4>
-          {execSummary(activeExecutorInfo, deadExecutorInfo)}
-        </div>
-      </div>
-      <div class = "row">
-        <div class="span12">
-          <h4>Executors</h4>
-          {execTable}
-        </div>
+      <div>
+        {
+        <div id="active-executors"></div> ++
+          <script src={UIUtils.prependBaseUri("/static/executorspage.js")}> </script>
+        }
       </div>;
 
-    UIUtils.headerSparkPage("Executors", content, parent)
+    UIUtils.headerSparkPage("Executors", content, parent, useDataTables = true)
   }
 
   /** Render an HTML row representing an executor */
```

Comment:
it seems like there is a bunch of other things in here that were used by render that can be removed. execRow, execSummaryRow, taskData, etc.